### PR TITLE
OpenSILEX local installation documentation improvements

### PR DESCRIPTION
**Pull request by Andreas-Garcia**
Some improvement after testing myself the local installation process :
- Corrections of English mistakes
- "PHIS" replaced by "OpenSILEX" exept for the names of the web projects and the names of the GitHub repositories because it has to be tested
-  Unification of the document formatting (file and folder names in code styles etc.)
- Improvements of the document formatting (e.g missing upper case letters added)
- Addition of precision on some parts (e.g when generating a key for GitHub)
- Miscellaneous improvements (e.g missing "sudo" in some commands, some file's path were the ones of the previous redactor but were not relevant for the reader)

### CODE REVIEW

File: docs/localInstallation.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Phis Deployment
+PostgreSQL# OpenSILEX Deployment
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]: did you mean "# OpenSILEX Deployment" ? without PostgreSQL

File: docs/localInstallation.md [hunk 2]
```diff
@@ -187,267 +200,278 @@ and add lines:
...
-to:
+by:
 ```XML
 <Connector port="8080" protocol="HTTP/1.1"
               connectionTimeout="20000"
               redirectPort="8443" />
 ```
-You have to allow **encoded slash** in **/home/tomcat/apache-tomcat/conf/catalina.properties**. If you have the line **"org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=false"**, change false value by true.  
-If you don't have this line add **"org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true"** at the end of file.
+You have to allow `encoded slash` in `/home/tomcat/apache-tomcat&lt;version>/conf/catalina.properties`. If you have the line `org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=false`, change `false` value by `true`.
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]: did you mean apache-tomcat<version> ?

File: docs/localInstallation.md (hunk 1 above)
Comment (Andreas-Garcia) [reply docs/localInstallation.md]: yes that is what I meant

File: docs/localInstallation.md (hunk 2 above)
Comment (Andreas-Garcia) [reply docs/localInstallation.md]: yes that is what I meant

File: docs/localInstallation.md [hunk 3]
```diff
@@ -187,267 +200,278 @@ and add lines:
...
+
+Get source from GitHub, directly from the OpenSILEX-webapp development repository:
 ```bash
-  cd ~Phis
-  git clone https://github.com/OpenSILEX/phis-webapp.git
+cd ~/OpenSILEX
+git clone https://github.com/OpenSILEX/phis-webapp.git
 ```
-Preferably, get the source from the last release at [phis-webapp/releases](https://github.com/OpenSILEX/phis-webapp/releases).
+Preferably, get the source from the last release at [OpenSILEX-webapp/releases](https://github.com/OpenSILEX/OpenSILEX-webapp/releases).
```
[the remark this answers is not in the document]
Comment (Andreas-Garcia) [reply docs/localInstallation.md]: yes it is indeed

File: docs/localInstallation.md [hunk 4]
```diff
@@ -191,267 +204,278 @@ and add lines:
...
+
+Get source from GitHub, directly from the `phis-ws` development repository:
 ```bash
-  cd ~Phis
-  git clone https://github.com/OpenSILEX/phis-ws.git
+cd ~/OpenSILEX
+git clone https://github.com/OpenSILEX/phis-ws.git
 ```
-Preferably, get the source from the last release at [phis-ws/releases](https://github.com/OpenSILEX/phis-ws/releases).
+Preferably, get the source from the last release at [opensilex-ws/releases](https://github.com/OpenSILEX/phis-ws/releases).
```
Comment (Gouga34) [inline docs/localInstallation.md]: `phis-ws/releases` The repository has not been renamed yet.

File: docs/localInstallation.md [hunk 5]
```diff
@@ -583,201 +608,180 @@ ws.layers.dir=/var/www/html/layers
...
-  - *webservices.php*  
-Adapt the last line with the correct URL in our case is:
-**http://127.0.0.1:8080/phis2ws/rest**
-
-If you are all exactly doing like in this document, it's the only configuration file you need to change. But you can check other configuration files: compare paths and URL with information written in web services configuration files.
-
-
+To deploy the web application in localhost, it isn't necessary to open it with Netbeans:
+- Edit `/var/www/html/phis-webapp/config/web_services.php`
+- Set the value of `WS_PHIS_PATH_DOC`to `http://127.0.0.1:8080/phis2ws/rest`
```
Comment (Gouga34) [inline docs/localInstallation.md]: the phisws must be the same in the line 632 (http://127.0.0.1:8080/phisws) and this line (http://127.0.0.1:8080/phis2ws/rest). Which one is the right url ?

File: docs/localInstallation.md [hunk 6]
```diff
@@ -583,201 +608,180 @@ ws.layers.dir=/var/www/html/layers
...
+Netbeans doesn't find your JDK installation. To fix it, check if you have indicated the correct JDK folder to Netbeans in the `~/netbeans-8.2/etc/netbeans.conf` file (line 57).
+If the error persists, your JDK installation may haven't been done correctly. Uninstall JDK and use [this procedure](#jdk) to reinstall.
 
+### Problems with Composer
 
-### Problems with composer
+If you have installed Composer with `apt` from Ubuntu packages, please uninstall Composer (consider deleting the cache and the configuration files in the `~/.cache` and `/etc` folders) and reinstall Composer with [this porcedure](#composer).
 
-If you are installed composer with apt from ubuntu packages, please uninstall composer (think to remove cache and configuration files in ~/.cache and /etc folders) and reinstall with [this porcedure](#composer).
+If doesn't fix the problem, please check [composer troubleshooting page](#https://getcomposer.org/doc/articles/troubleshooting.md).
```
Comment (Gouga34) [inline docs/localInstallation.md]: remove # : #https://getcomposer.org/doc/articles/troubleshooting.md

File: docs/localInstallation.md (hunk 5 above)
Comment (Andreas-Garcia) [reply docs/localInstallation.md]: I would say http://127.0.0.1:8080/phisws

Comment (Gouga34) [reply docs/localInstallation.md]: Than it might be this url everywhere ;)

File: docs/localInstallation.md [hunk 7]
```diff
@@ -19,170 +19,183 @@ layout: default
...
+
+This document explains you how to deploy OpenSILEX on your personnal computer.  
+In this document, commands are for **Ubuntu 16.04**. However, the majority of these commands are compatible with all Debian distributions which have the package manager Aptitude.
 
 ## Prerequisite
 
 ### Software
 
-#### Mongodb + robo3t
+#### MongoDB + Robo 3T
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]: Possible conflict with 398c2f3e4c99094cde24e5d1ff2ed19be9914585 (it's true for the other modified headers also)

File: docs/localInstallation.md [hunk 8]
```diff
@@ -19,170 +19,183 @@ layout: default
...
     - [Errors with the web service](#errors-with-the-web-service)
     - [Errors with the web application](#errors-with-the-web-application)
-    - [Other problems concerned webapp and web service](#other-problems-concerned-webapp-and-web-service)
+    - [Other problems concerning the web application and the web service](#other-problems-concerned-webapp-and-web-service)
 
 ## Introduction  
-This document explains you how to deploy PHIS on your personnal computer.  
-In this document commands lines are for **Ubuntu 16.04**, but for the majority of them, they are compatible with all Debian distribution which have the package manager Aptitude.
+
+This document explains you how to deploy OpenSILEX on your personnal computer.
```
[the remark this answers is not in the document]
Comment (Andreas-Garcia) [reply docs/localInstallation.md]: I'll change it to OpenSILEX PHIS

File: docs/localInstallation.md [hunk 9]
```diff
@@ -214,250 +246,266 @@ by:
...
+git clone https://github.com/OpenSILEX/ontology-vocabularies
 ```
-Preferably, get the source from the last release at [ontology-phis-oepo-field/releases](https://github.com/OpenSILEX/ontology-phis-oepo-field/releases).
+Preferably, get the source from the last release at [ontology-vocabularies/releases](https://github.com/OpenSILEX/ontology-vocabularies/releases).
 
 #### Database file
 
-Download the database dump file [phis_st_dump.sql](assets/phis_st_dump.sql).
-
+Download the database dump file [opensilex_st_dump.sql](docs-community-dev/docs/assets/phis_st_dump).
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]:
```suggestion
Download the database dump file [opensilex_st_dump.sql](docs-community-dev/docs/assets/opensilex_st_dump).
```

File: docs/localInstallation.md [hunk 10]
```diff
@@ -214,250 +246,266 @@ by:
...
+
+Download the dump file to import [here](opensilex_st_dump.sql) (make shure you download it in a folder where you are fully owner - like the `/var/lib/postgresql/` folder - because of PosgreSQL ownership issue when importing data).
+
+Import data with :
 ```bash
-  psql -U phis diaphen < ~/Phis/phis_st_dump.sql
+psql -U opensilex -h 127.0.0.1 <experimental_installation_name> < /var/lib/postgresql/opensilex_st_dump.sql
 ```
-You can find [dump file](assets/phis_st_dump.sql).
+You can find [dump file](opensilex_st_dump.sql).
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]:
```suggestion
 You can find [dump file](assets/opensilex_st_dump.sql).
```

File: docs/localInstallation.md [hunk 11]
```diff
@@ -214,250 +246,266 @@ by:
...
+git clone https://github.com/OpenSILEX/ontology-vocabularies
 ```
-Preferably, get the source from the last release at [ontology-phis-oepo-field/releases](https://github.com/OpenSILEX/ontology-phis-oepo-field/releases).
+Preferably, get the source from the last release at [ontology-vocabularies/releases](https://github.com/OpenSILEX/ontology-vocabularies/releases).
 
 #### Database file
 
-Download the database dump file [phis_st_dump.sql](assets/phis_st_dump.sql).
-
+Download the database dump file [opensilex_st_dump.sql](docs-community-dev/docs/assets/opensilex_st_dump).
```
Comment (pierreetiennealary) [inline docs/localInstallation.md]:
```suggestion
Download the database dump file [assets/opensilex_st_dump).
```

Comment (pierreetiennealary) [inline docs/localInstallation.md]:
```suggestion
Download the database dump file [opensilex_st_dump.sql](assets/opensilex_st_dump).
```

**Pull request merged**